### PR TITLE
youtube-dl: bump version and use precompiled script

### PR DESCRIPTION
**Pull request by ianchi**
Update to version 2017.06.05
Use precompiled scripts, in line with the new Python package. (faster run time, with the downside of a bit larger pkg size)

Signed-off-by: Adrian Panella <ianchi74@outlook.com>

Maintainer: me 
Compile tested: ipq806x 
Run tested: ipq806x

Description:

### CODE REVIEW

File: multimedia/youtube-dl/Makefile [hunk 1]
```diff
@@ -8,18 +8,19 @@
...
 
 PKG_NAME:=youtube-dl
-PKG_VERSION:=2017.02.17
+PKG_VERSION:=2017.06.05
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://yt-dl.org/downloads/$(PKG_VERSION)/
-PKG_MD5SUM:=d38dbe099af22eb9af35b0e8b274dc4e
+PKG_MD5SUM:=1f9b35f2c767374967d0afef08132fe2
```
Comment (diizzyy) [inline multimedia/youtube-dl/Makefile]: MD5SUM is deprecated, use PKG_HASH (SHA256) instead.

**Comment by diizzyy** [comment]
@hnyman 
Please don't commit more MD5SUM stuff, there's work being done to clean this up before merging and it's counterproductive.

**Comment by ianchi** [comment]
@diizzyy added switch to PKG_HASH

**Comment by diizzyy** [comment]
LGTM

**Pull request merged**